src/bounding_volume/bounding_volume.rs

```diff
@@ -5,6 +5,8 @@ use na::RealField;
 pub trait HasBoundingVolume<N: RealField, BV> {
     /// The bounding volume of `self` transformed by `m`.
     fn bounding_volume(&self, m: &Isometry<N>) -> BV;
+    /// The bounding volume of `self`.
+    fn local_bounding_volume(&self) -> BV;
```

Comment:
To avoid a breaking change:

```suggestion
    fn local_bounding_volume(&self) -> BV {
        self.bounding_volume(&Isometry::identity())
    }
```